PinpointKit/PinpointKit/Sources/ScreenshotCell.swift

```diff
@@ -100,9 +101,22 @@ class ScreenshotHeaderView: UIView {
         screenshotButton.layer.borderColor = tintColor.CGColor
     }
     
-    // MARK: - ScreenshotHeaderView
+    override func addSubview(_ view: UIView) {
+        // Prevents the adding of separators to this cell.
+        let separatorHeight = 1.0 / UIScreen.mainScreen().scale
```

Comment:
I think it would be nice to have this be an extension on `UIScreen`. Something like `pixelHeight`.